modules/ROOT/pages/deploy-to-cloudhub.adoc

```diff
@@ -0,0 +1,148 @@
+      <environment>${environment}</environment>
+      <properties>
+        <key>value</key>
+      </properties>
+    </cloudHubDeployment>
+  </configuration>
+</plugin>
+----
++
+. From the command line in your project's folder, package the app and execute the deploy goal:
++
+[source,console,linenums]
+----
+mvn clean package deploy -DmuleDeploy
+----
+
+== Redeploy to CloudHub
+
+To redeploy a Mule application using Mule Maven plugin, run the same command as you did to deploy. +
+CloudHub rewrites the app you had deployed.
+
+=== Undeploy from CloudHub
+
+To undeploy a Mule application using Mule Maven plugin, run the following command:
+
+[source,console,linenums]
+----
+mvn mule:undeploy
+----
+
+The undeploy command also deletes the app in Mule Maven plugin 3.3.0 and later versions.
+
+=== CloudHub Deployment Reference
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Parameter | Description
+|`cloudHubDeployment` | Top-Level Element.
+| `uri` | Your Anypoint Platform URI. +
+If not set, by default this value is set to +https://anypoint.mulesoft.com+.
+| `muleVersion` | The Mule version that will run in your CloudHub instance. +
+You need to ensure that this value is equal to or higher than the minimum required Mule version of your application.
+| `username` | Your CloudHub username.
+| `password` | Your CloudHub password.
+| `applicationName` | The name of your application in CloudHub. +
+This name is part of the domain of your deployed app. For example, naming your application application-1 makes your app's public domain +application-1.cloudhub.io+.
+| `artifact` | Absolute file location of the JAR to be deployed. +
+If not set, by default it is the location of the JAR generated at the package phase
+| `environment` | The CloudHub environment to which you want to deploy. This value must match any environment configured in your CloudHub account. +
+For Example:
+[source,xml,linenums]
+----
+<environment>Sandbox</environment>
+----
+include::partial$mmp-concept.adoc[tag=propertiesParameterDescription]
+| `workers` | The number of workers. +
+By default, it is set to 1.
+| `workerType` | Size of each worker. +
+
+Accepted values are:
+
+MICRO (0.1 vCores) +
+SMALL (0.2 vCores) +
+MEDIUM (1 vCore ) +
+LARGE (2 vCores) +
+XLARGE (4 vCores) +
+XXLARGE (8 vCores) +
+4XLARGE (16 vCores)
+
+The default value is MICRO.
+| `region` | Region of worker clouds.
+
+Accepted values are:
+
+* us-east-1      -  US East (N. Virginia)
+* us-west-2      -  US West (Oregon)
+* eu-west-1      -  EU (Ireland)
+* ap-southeast-2 -  Asia Pacific (Sydney)
+* ap-southeast-1 -  Asia Pacific (Singapore)
+* us-west-1      -  US West (N. California)
+* eu-central-1   -  EU (Frankfurt)
+* ap-northeast-1 -  Asia Pacific (Tokyo)
+* eu-west-2      -  EU (London)
+* ca-central-1   -  Canada (Central)
+* sa-east-1      -  South America (São Paulo)
+* us-east-2      -  US East (Ohio)
+
+The default value is  us-east-1.
```

Comment:
You might want to add below this table some new reference concepts ObjectStoreV2 and persistenQueues https://github.com/mulesoft/docs-mule-runtime/pull/1021#pullrequestreview-32324486